crawl-ref/source/dat/des/arrival/small.des

```diff
@@ -3412,3 +3412,126 @@ MAP
   ...........
    .........
 ENDMAP
+
+##############################################################
+NAME:    heliophobic_arrival_battle_scene
+TAGS:    arrival no_monster_gen no_trap_gen no_pool_fixup
+ORIENT:  float
+ITEM:    human skeleton / dwarf skeleton , \
+         goblin skeleton / gnoll skeleton / \
+         kobold skeleton / orc skeleton
+SHUFFLE: cx
+SHUFFLE: tvw
+FTILE:   .{det+ = floor_marble
+MAP
+cccc+cccc
+ce..de.ec
+c.e.w..ec
+cd..we..c
+c.d.w.d.c
+c..e..e.c
+ced.{d.ec
+c.d..de.c
+ccccccccc
+ENDMAP
+
+##############################################################
+NAME:    heliophobic_arrival_enchanted_forest
+TAGS:    arrival no_monster_gen no_trap_gen
+ORIENT:  float
+SHUFFLE: TAG
+FTILE:  .{TAGtF = floor_moss
+MARKER:  F = lua:fog_machine{ \
+             pow_max = 20, delay_min = 20, delay_max = 200, \
+             size = 5, start_clouds = 4, \
+             cloud_type = "black smoke"}
```

Comment:
```suggestion
             cloud_type = "thin mist"}
```
```suggestion
             cloud_type = "magical condensation"}
```

These two cloud types (either pick one or randomize) are a bit more in line with enchanted forests!

Reply:
Got it!  

I just thought something more LOS-breaking would be more interesting tactically.